src/structures/VoiceState.js

```diff
@@ -63,10 +64,22 @@ class VoiceState extends Base {
      */
     this.streaming = data.self_stream || false;
     /**
-     * The ID of the voice channel that this member is in
+     * The ID of the voice channel or stage channel that this member is in
      * @type {?Snowflake}
      */
     this.channelID = data.channel_id || null;
+    /**
+     * Whether this member is suppressed from speaking. This property is specific to stage channels only.
+     * If true, the user is unable to speak.
+     * @type {boolean}
+     */
+    this.suppress = data.suppress;
+    /**
+     * The time at which the member requested to speak. This property is specific to stage channels only.
+     * @type {?number}
+     */
+    this.requestToSpeakTimestamp =
+      typeof data.request_to_speak_timestamp === 'string' ? new Date(data.request_to_speak_timestamp).getTime() : null;
```

Comment:
```suggestion
    this.requestToSpeakTimestamp =
      data.request_to_speak_timestamp ? new Date(data.request_to_speak_timestamp).getTime() : null;
```
`typeof` isn't necessary in this case